sql/mozfun/glam/histogram_fill_buckets/udf.sql

```diff
@@ -18,11 +18,7 @@ RETURNS ARRAY<STRUCT<key STRING, value FLOAT64>> AS (
         SAFE_CAST(key AS STRING) = e.key
     )
     SELECT
-      ARRAY_AGG(
-        STRUCT<key STRING, value FLOAT64>(SAFE_CAST(key AS STRING), value)
-        ORDER BY
-          CAST(key AS int64)
-      )
+      ARRAY_AGG(STRUCT<key STRING, value FLOAT64>(SAFE_CAST(key AS STRING), value) ORDER BY key)
```

Comment:
Is it ever possible that key isn't a string and the cast is needed?  I don't think it is because the parameter is a string but maybe there's a reason it's here.  Fine either way, I'm just curious.

Reply:
If the input_map key type were `ANY`, this would be needed, but so far I don't think this has been the case. I'll simplify this a bit.